clients/cobol-lsp-vscode-extension/src/__tests__/JavaCheckTest.ts

```diff
@@ -27,8 +28,73 @@ describe("Check Java version", () => {
         expect(JavaCheck.isJavaVersionSupported('openjdk version "12.0.1" 2018-10-16')).toBeTruthy();
         expect(JavaCheck.isJavaVersionSupported("java 11 2018-09-25")).toBeTruthy();
         expect(JavaCheck.isJavaVersionSupported("java 12 2019-03-19")).toBeTruthy();
+        expect(JavaCheck.isJavaVersionSupported("java 13 2019-09-17")).toBeTruthy();
+        expect(JavaCheck.isJavaVersionSupported("java 14 2020-03-17")).toBeTruthy();
     });
-    it("checks if Java version is not supported", async () => {
+
+    it("when version is not supported", async () => {
         expect(JavaCheck.isJavaVersionSupported('java version "1.7.0_131"')).toBeFalsy();
+        expect(JavaCheck.isJavaVersionSupported('java version "1.5.0_22"')).toBeFalsy();
+    });
+});
+
+describe("Checks Java installation", () => {
+
+    let javaCheck: JavaCheck;
+    let map: any;
+    let checkFn: any;
+    let stderrFn: any;
+    const expectedErrMsgSupportedJavaVersion = "Java version 8 expected";
+    const expectedErrMsgJavaVersionNotFound = "Java 8 is not found";
```

Comment:
Are we supporting java 8 still?

Reply:
We will update that outdated message with "Minimum expected Java version is 8" in service class.

Reply:
Oh, sorry, I miss that one `map = {};`